pkg/virtctl/create/clone/clone.go

```diff
@@ -271,15 +291,15 @@ func (c *createClone) typeToTypedLocalObjectReference(sourceOrTargetType, source
 	}
 
 	switch sourceOrTargetType {
-	case "vm", "VM", "VirtualMachine", "virtualmachine":
-		kind = "VirtualMachine"
+	case "vm", "VM", virtualMachineKind, "virtualmachine":
+		kind = virtualMachineKind
 		apiGroup = "kubevirt.io"
-	case "snapshot", "VirtualMachineSnapshot", "vmsnapshot", "VMSnapshot":
+	case "snapshot", virtualMachineSnapshotKind, "vmsnapshot", "VMSnapshot":
 		if !isSource {
 			return nil, generateErr()
 		}
 
-		kind = "VirtualMachineSnapshot"
+		kind = virtualMachineSnapshotKind
 		apiGroup = "snapshot.kubevirt.io"
```

Comment:
Looks inconsistent to me, did the linter complain only about the two literals you've added consts for?